whois-api/src/main/java/net/ripe/db/whois/api/httpserver/dos/AbstractDoSFilterHolder.java

```diff
@@ -0,0 +1,41 @@
+package net.ripe.db.whois.api.httpserver.dos;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class AbstractDoSFilterHolder extends FilterHolder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractDoSFilterHolder.class);
+
+    AbstractDoSFilterHolder(final boolean dosFilterEnabled, final String trustedIpRanges){
+        if (!dosFilterEnabled) {
+            LOGGER.info("DoSFilter is *not* enabled");
```

Comment:
Found a better way. just logging enough , setInitParameter will take care of rest

Reply:
that explain why we didn't have the return in master when is disabled

Reply:
done, I rely on that parameter